package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.0.0",
   "description": "Streamlined DOM Events for most.js",
   "main": "dist/dom-event.js",
+  "typings": "type-definitions/dom-event.d.ts",
```

Comment:
Since the lib gets built into a single file and only that file is published to npm, we probably need to either 1) also add `type-definitions/*.ts` to `files` below, or 2) keep the type definitions in `dist` alongside the built file, and change `files` to include `dist/*`.

Any thoughts on which of those seems better?